src/content/reference/manifest.mdx

```diff
@@ -509,6 +509,41 @@ metadata:
     custom.label/dev: "true"
 ```
 
+#### mode (string, optional)
+
+Defines how `okteto up` will setup the development container and behave within the cluster.
+
+Currently two options are available:
+- `sync`: (default) Sync & run your code in a remote development container.
+- `hybrid`: Run your code on your local computer instead of in a remote development container, while rest of your service components runs in your remote cluster.
+
+Most use-cases should use default `sync` mode, but `hybrid` may be desired to achieve a faster development experience in some scenarios:
+
+|   | `sync` | `hybrid` |
+|---|---|---|
+| Soure code  | Uses Syncthing to sync files bidirectionally with your remote development container | Uses local source code directly, no files syncing needed |
+| `command` context | Runs in the _remote development container_ | Runs on your _local machine_, so traffic need to be rerouted from/to cluster, e.g. using `reverse` option |
+
+##### `hybrid` motivation and example
+
+A typical use-case and motivation for using `hybrid` is when developing a service that require "very fast" file-change-to-reload times. Frontend development tooling (e.g. Webpack) is a common example. Since hybrid mode avoids having to sync file changes to remote development container and wait for the container to detect file changes to trigger a rebuild, it's typically order-of-magnitude faster than `sync` mode.
```

Comment:
Do we have data to back this up? If I read this, my first reaction would be never using "sync" mode. Why would I pick something 10 times slower?